cmd/zync/from-kafka/command.go

```diff
@@ -1,4 +1,4 @@
-package sync
+package from_kafka
```

Comment:
I'd love to stay away from hyphens and underscores in Go file and package names.
```suggestion
package fromkafka
```